src/pages/resultsView/coExpression/CoExpressionViz.tsx

```diff
@@ -304,7 +334,8 @@ export default class CoExpressionViz extends React.Component<ICoExpressionVizPro
                                 showMutations={this.plotShowMutations}
                                 logScale={this.plotLogScale}
                                 handlers={this.props.plotHandlers}
-                                molecularProfile={this.props.molecularProfile}
+                                queryMolecularProfile={this.props.queryProfile}
+                                subjectMolecularProfile={this.props.subjectProfile}
```

Comment:
I'm not really getting the conceptual difference between query and subject, besides as explaining to the user. Seems to me it could more clearly be `xMolecularProfile` and `yMolecularProfile`? Could you please change it to that everywhere, unless I'm misunderstanding?

Reply:
I've changed all the references to 'subjectProfiles' into 'xProfiles', and 'queryProfiles' to 'yProfiles' - that's how they are displayed in the plot.